cli/command/registry/login.go

```diff
@@ -120,7 +120,12 @@ func runLogin(ctx context.Context, dockerCli command.Cli, opts loginOptions) err
 }
 
 func loginWithStoredCredentials(ctx context.Context, dockerCli command.Cli, authConfig registrytypes.AuthConfig) (msg string, _ error) {
-	_, _ = fmt.Fprintf(dockerCli.Out(), "Authenticating with existing credentials...\n")
+	_, _ = fmt.Fprintf(dockerCli.Out(), "Authenticating with existing credentials...")
+	if authConfig.Username != "" {
+		_, _ = fmt.Fprintf(dockerCli.Out(), " [Username: %s]", authConfig.Username)
+	}
+
+	_, _ = fmt.Fprintf(dockerCli.Out(), "\nTo login with a different account, run 'docker logout' followed by 'docker login'\n\n")
```

Comment:
This is clearer for sure! I wish we could improve this eventually. Maybe in the future we can do something like prompt the user, if TTY – something like `Found credentials for index.docker.io. Reuse credentials (Y/n)?`

WDYT @thaJeztah @vvoland ?